src/Provider/GoogleMaps/GoogleMaps.php

```diff
@@ -267,6 +269,19 @@ private function updateAddressComponent(AddressBuilder $builder, string $type, $
                 $builder->addAdminLevel(intval(substr($type, -1)), $values->long_name, $values->short_name);
                 break;
 
+            case 'sublocality_level_1':
+            case 'sublocality_level_2':
+            case 'sublocality_level_3':
+            case 'sublocality_level_4':
+            case 'sublocality_level_5':
+                $newSubLocalityLevel = [
+                    'level' => intval(substr($type, -1)),
+                    'name' => $values->long_name,
+                    'code' => $values->short_name,
+                ];
+                $builder->setValueInArray('subLocalityLevel', $newSubLocalityLevel);
```

Comment:
I had similar idea, but $newSubLocalityLevel[] will make array with key 0 each time
`$newSubLocalityLevel = [0=> [ 
                                                 'level' => 3,
                                                 'name' => 'name'
                                                 'code' => 'code'
                                                 ]
                                         ]`

and `setValue(..)` will pass this array, and replace previous value (which also had 0 key). 
If we call  `setValue(..)`  after the switch statement like that, we would override `subLocalityLevel` subarray each time we call the function. 
Maybe I missed out on something?